core/src/main/scala/org/scalarelational/datatype/ObjectSerializationConverter.scala

```diff
@@ -12,9 +13,9 @@ import org.scalarelational.model.Datastore
  * @author Matt Hicks <matt@outr.com>
  */
 class ObjectSerializationConverter[T <: AnyRef] extends DataType[T] {
-  override def jdbcType = Types.BINARY
+  override def jdbcType = Types.BLOB
```

Comment:
Because MariaDB has a maximum row size. A `BINARY` column adds to the used row size. A `BLOB` does not. It works for H2 as they have no limit, but with MariaDB we could not serialize abitrary objects and add them to a row. A consequence would be, that in some cases it works (sum of all objects < row_size_limit) and in some it misteriously does not.